dotCMS/src/main/webapp/html/portlet/ext/folders/edit_folder.jsp

```diff
@@ -166,19 +166,19 @@ dojo.require("dotcms.dojo.data.StructureReadStore");
 						<dt><%= LanguageUtil.get(pageContext, "Path-To-Folder") %>:</dt>
 						<dd>
 							<% if (parentFolder != null) { %>
-								<input id="pathToFolder" readonly="true" style="width:250px;border:0;" value="<%= APILocator.getIdentifierAPI().find(parentFolder).getPath() %><%= folder.getName() %>" />
+								<input id="pathToFolder" readonly="true" style="width:1000px;border:0;" value="<%= APILocator.getIdentifierAPI().find(parentFolder).getPath() %><%= folder.getName() %>" />
 							<% } else { %>
-								<input id="pathToFolder" readonly="true" style="width:250px;border:0;" value="/<%= folder.getName() %>" />
+								<input id="pathToFolder" readonly="true" style="width:1000px;border:0;" value="/<%= folder.getName() %>" />
 							<% } %>
 							<!--<html:hidden property="name" />-->
 						</dd>
 					<%}else{%>
 						<dt><%= LanguageUtil.get(pageContext, "Path-To-Folder") %>:</dt>
 							<dd>
 								<% if (parentFolder != null) { %>
-									<input id="pathToFolder" readonly="true" style="width:250px;border:0;" value="<%= APILocator.getIdentifierAPI().find(parentFolder).getPath() %>" />
+									<input id="pathToFolder" readonly="true" style="width:1000px;border:0;" value="<%= APILocator.getIdentifierAPI().find(parentFolder).getPath() %>" />
 								<% } else { %>
-									<input id="pathToFolder" readonly="true" style="width:250px;border:0;" value="/" />
+									<input id="pathToFolder" readonly="true" style="width:1000px;border:0;" value="/" />

```

Comment:
I think is better do it with percentages and max-width, somethink like:

https://github.com/dotCMS/core/pull/17862/files

Reply:
Done